version/version.go

```diff
@@ -23,7 +23,7 @@ var (
 	Package = "github.com/containerd/containerd"
 
 	// Version holds the complete version number. Filled in at linking time.
-	Version = "1.6.17+unknown"
+	Version = "1.6.18+unknown"
```

Comment:
Oh! I keep forgetting to ask; is there a reason we always use "+unknown" for the actual released tag? Should it be without (and the +unknown added back after that?)

Reply:
I always assumed this was to clearly show a binary that was built outside the `Makefile` process (e.g. `go install`?) so that you can differentiate it from an official build via the `Makefile`; if you build via the `Makefile` then this whole string is replaced with:
```
VERSION ?= $(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
```

Reply:
I wasn't sure what the reason was; I know runc uses +dev after releasing (release PR contains 2 commits, and the first one is tagged).

Currently, I think code using containerd as a dependency gets the +unknown (which made me look a few times if I was actually using the tagged relese), which also is included in the user agent (as default, that is)